carts/views.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from django.http           import JsonResponse
+from django.views          import View
+from .models               import Carts
+from product.models        import Products
+from users.utils           import login_decorator
+
+class CartView(View):
+    @login_decorator
+    def post(self, request):
+        data = json.loads(request.body)
+        user = request.user
+
+        Carts.objects.create(
+            user = user,
+            product = Products.objects.get(id=data['product']),
+            quantity = data['quantity']
+        )
+        return JsonResponse({'message': 'added_to_cart'}, status=201)
+
+    @login_decorator
+    def get(self, request):
+        user = request.user
+        user_cart = Carts.objects.filter(user=user)
```

Comment:
변수는 간결하고 명확하게 만들어주세요. user라는 용어는 too much!
```suggestion
       carts = Carts.objects.filter(user=user)
```

Reply:
user를 그만 쓰라는건가요?? 아니면 user_cart??